### PR TITLE
[Bug] [Doc] Fix e2e-test dead link

### PR DESCRIPTION
**Pull request by QuakeWang**
close: https://github.com/apache/dolphinscheduler/issues/8535

### CODE REVIEW

**Review by SbloodyS — DISMISSED** [review]
LGTM

**Comment by SbloodyS** [comment]
@zhongjiajie Please review. Thanks.

**Comment by zhongjiajie** [comment]
😭  I remind that it maybe not work in https://github.com/apache/dolphinscheduler-website/pull/699#discussion_r811746689

File: community/en-us/join/e2e-guide.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ In each issue, we list the content to be tested and the expected results, which
 
 ### How to write test code?
 
-After taking up the corresponding task, the next step is to get to the crux of writing the code. Many partners may not be familiar with E2E test codes, so you can refer to this page: [e2e-test](../../../development/zh-cn/e2e-test.md).
+After taking up the corresponding task, the next step is to get to the crux of writing the code. Many partners may not be familiar with E2E test codes, so you can refer to this page: [e2e-test](../../../en-us/development/e2e-test.md).
```
Comment (zhongjiajie) [inline community/en-us/join/e2e-guide.md:32]: I take a quick look, our document will change the path when we build from source. maybe we should add full http when we need to link between `community` and `developer` tab, just like `[Contributing Guides](https://dolphinscheduler.apache.org/en-us/community/development/contribute.html)` in https://dolphinscheduler.apache.org/en-us/community/team.html

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

File: community/en-us/join/e2e-guide.md (hunk 1 above)
Comment (zhongjiajie) [reply community/en-us/join/e2e-guide.md:32]: I mean, we should use the link `[e2e-test](https://dolphinscheduler.apache.org/en-us/development/e2e-test.html)`

**Comment by QuakeWang** [comment]
> 😭 I remind that it maybe not work in [#699 (comment)](https://github.com/apache/dolphinscheduler-website/pull/699#discussion_r811746689)


That is my bad, I only checked the link of pull request, and the link of e2e-test was forgotten.

File: community/en-us/join/e2e-guide.md (hunk 1 above)
Comment (QuakeWang) [reply community/en-us/join/e2e-guide.md]: > I mean, we should use the link `[e2e-test](https://dolphinscheduler.apache.org/en-us/development/e2e-test.html)`

I have updated it.

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**